.travis.yml

```diff
@@ -26,10 +26,6 @@ addons:
 before_script:
   - export DISPLAY=:99.0; sh -e /etc/init.d/xvfb start ;
   - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost \&  ;
-before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.2.1
```

Comment:
But "we" need to update the [documentation](https://github.com/theia-ide/theia/blob/master/doc/Developing.md#prerequisites) and adjust the `yarn` version.

Reply:
In general Yarn is backwards compatible; I can update the docs if you think it worth it.

Reply:
Yes, please. And pin a working version. I would like to avoid odd situations when the CI has a different yarn version than one of the local machines and only one of them works. Thanks!